Error: Cannot find module 'debug'

Hello, I see this error when using postgraphile as a library and including pg-many-to-many

I have `"@graphile-contrib/pg-many-to-many": "^1.0.0-beta.4",` and 
`"postgraphile": "^4.4.0-beta.13",`

```sh
Error: Cannot find module 'debug'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/task/node_modules/@graphile-contrib/pg-many-to-many/src/PgManyToManyRelationPlugin.js:1:84)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

--- a/src/PgManyToManyRelationPlugin.js
+++ b/src/PgManyToManyRelationPlugin.js
@@ -1,6 +1,3 @@
-const debugFactory = require("debug");
-const debug = debugFactory("graphile-build-pg");
-
 module.exports = function PgManyToManyRelationPlugin(
   builder,
   { pgSimpleCollections }
@@ -119,12 +116,11 @@ module.exports = function PgManyToManyRelationPlugin(
             null
           );
           if (!RightTableType) {
-            debug(
+            throw new Error(
               `Could not determine type for table with id ${
                 junctionRightConstraint.classId
               }`
             );
-            return memo;
           }
           const RightTableConnectionType = getTypeByName(
             inflection.connection(RightTableType.name)

